internal/statestore/redis.go

```diff
@@ -82,7 +82,12 @@ func newRedis(cfg config.View) Service {
 		MaxActive:   cfg.GetInt("redis.pool.maxActive"),
 		IdleTimeout: cfg.GetDuration("redis.pool.idleTimeout"),
 		Wait:        true,
-		TestOnBorrow: func(c redis.Conn, _ time.Time) error {
+		TestOnBorrow: func(c redis.Conn, t time.Time) error {
+			// Check if this connection have been returned for more than 15 seconds, if so, do a PING check.
```

Comment:
This comment isn't clear without context.  What does "this connection have been returned for more than 15 seconds" mean precisely?